README.md

```diff
@@ -1493,6 +1493,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 - [url-shortener](https://github.com/cagataycali/url-shortener) `⚠` - Shitty url shortener, emoji and AI powered. `MIT` `Nodejs`
 - [URL-Shortener](https://github.com/sapioit/URL-shortener) - Make your long links short and only uses lowercase, so you can write and spell them faster. ([Demo](http://scurtez.cf)) `GPL-3.0` `PHP`
 - [YOURLS](http://yourls.org/) - YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener. Features include password protection, URL customization, bookmarklets, statistics, API, plugins, jsonp. ([Source Code](https://github.com/YOURLS/YOURLS)) `MIT` `PHP`
+[URL Shorter](https://gitlab.jonasled.de/jonasled/url_shorter_docker) - A easy to use URL shorter, written in Python with Flask and docker. [demo](https://kurz.ml) `GPL-3.0` `Docker/Python`
```

Comment:
Change Docker/Python to just Python as this can be run without docker.

Reply:
If a Docker-based install is officially supported it does not harm to mention it? I think. However I think it should be the other way around (`Python/Docker`) as it is not *required*